ReactWindows/ReactNative/Modules/Dialog/DialogModule.cs

```diff
@@ -165,13 +165,29 @@ private void ShowDialog(MessageDialogInfo dialog)
                     }
 
                     await dialog.MessageDialog.ShowAsync();
-                });
+                }
+                ), dialog.RootViewHint);
             }
             catch (InvalidOperationException)
             {
                 // RootViewHint is bogus, i.e. no Dispatcher thread was found
                 dialog.ErrorCallback($"Alert failed: RootViewHint {dialog.RootViewHint} can't be mapped to a CoreApplicationView");
             }
         }
+
+        class UIBlock : IUIBlock
```

Comment:
Same as https://github.com/Microsoft/react-native-windows/blob/master/ReactWindows/ReactNative.Shared/Animated/NativeAnimatedModule.cs#L435-L448

Should we just create a default `UIBlock` class and share?

Reply:
I was thinking originally, but then I said nah, this is not a real class, it's more or less a pattern of calling. In other words too small to consider a refactoring.
Not a strong opinion though. We may refactor later if it becomes burdensome.